lib/StateMachineSkill.js

```diff
@@ -34,14 +34,28 @@ class StateMachineSkill extends AlexaSkill {
       return this.requestHandlers.IntentRequest(alexaEvent, reply);
```

Comment:
why have this as a separate piece? maybe we should also put the LaunchRequest handling with the rest of `requestToTurnIntoIntent`

Reply:
Done!